### DISCUSSION TITLE
Remove spaces and empty lines around the title.

### DISCUSSION THREAD

**Discussion by lesovsky** [charmbracelet/bubbletea#166] [Q&A] [answered]
Hi, trying to play with examples, and cannot remove spaces on the left of title and empty line between title and items in the [list-simple](https://github.com/charmbracelet/bubbletea/tree/master/examples/list-simple).

Tried to do this with Padding* and Margin* methods, but have no success. Is it possible to remove spaces and empty lines around the title?

**Comment by muesli** [comment, marked as answer]
The margins and paddings are defined at the top of the example: see `titleStyle` and `itemStyle`.
